docs/emigrate-nobles.rst

```diff
@@ -0,0 +1,36 @@
+emigrate-nobles
+===============
+
+.. dfhack-tool::
+    :summary: Allow inherited nobles to emigrate from fort to their site of governance.
+    :tags: fort units
+
+Tired of inherited nobility freeloading off your fortress making inane demands? Use this tool
+to have them (willingly) emigrate to their rightful lands.
+
+The unit must not be assigned to a squad.
```

Comment:
is this a limitation, or is the script interpreting squad assignment as an indicator that the player wants to keep the noble? I think we can unassign from squads programmatically.

Reply:
It was a limitation as I had not implemented that logic yet. I'll see what I can do (probably reference stuck-squad and uniform-unstick).